File: solutions/2881-split-strings-by-separator/solution.py
class Solution(object):
    def splitWordsBySeparator(self, words, separator):
        """
        :type words: List[str]
        :type separator: str
        :rtype: List[str]
        """
        l1=separator.join(words)
        return [item for item in l1.split(separator) if item]

File: solutions/2881-split-strings-by-separator/solution.java
class Solution {
    public List<String> splitWordsBySeparator(List<String> words, char separator) {
        List<String> ls=new ArrayList<>();
        String sep="\\"+String.valueOf(separator);
        for(String a:words)
        {
            String[] b=a.split(sep);
            for(String c:b)
                if(!c.isEmpty())
                    ls.add(c);
        }
        return ls;
    }
}
